app/components/images/imagesController.js

```diff
@@ -59,7 +59,28 @@ function ($scope, $state, Config, Image, ImageHelper, Messages, Pagination) {
     });
   };
 
-  $scope.removeAction = function () {
+  $scope.confirmRemovalAction = function (force) {
+    ModalService.confirm({
```

Comment:
I offer you this new solution in https://github.com/portainer/portainer/pull/562/commits/a5944dcb6ae066bbf47e52edd2075051b500cafe, based on object passing.